src/lib/get-topper-settings.js

```diff
@@ -98,12 +98,18 @@ const usePackageTopper = (content) => {
 	};
 };
 
-const useEditoriallySelectedTopper = (content) => {
+const useEditoriallySelectedTopper = (content, flags) => {
 	let backgroundColour;
+	let myFtButtonVariant = 'standard';
 
 	// Convert old palette colours to new palette colours from Methode
 	if (content.topper.layout === 'full-bleed-offset') {
 		backgroundColour = 'wheat';
+	} else if (content.topper.layout === 'deep-landscape') {
```

Comment:
Nitpick: the code on this file is getting complicated to read. Can you please add some tests for the part you are adding? It will help refactoring.

Reply:
Added in some extra tests to cover this